demos/main/src/main/AndroidManifest.xml

```diff
@@ -34,6 +34,7 @@
       android:banner="@drawable/ic_banner"
       android:largeHeap="true"
       android:allowBackup="false"
+      android:networkSecurityConfig="@xml/network_security_config"
       android:name="com.google.android.exoplayer2.demo.DemoApplication"
```

Comment:
Humble suggestion: I think it would be great for the reviewer to split into 2 pull requests:
- One for the Renderer change.
- One for the demo app intent change.

A third one would be allowing clear text in the demo app, but i suspect that's less likely to be accepted, since it was probably an explicit decision not to allow it.

This way, the decision of bringing in the tunneling fix is not affected by the decision of enabling clear text, plus avoids any back-and-forth.

Reply:
thanks, the demo app change was not really intended to submit.  I'll pull it out.

Reply:
This change is now removed from the branch.